scripts/generate_user_space_activities_counters.ts

```diff
@@ -0,0 +1,45 @@
+import 'dotenv/config';
+import { refreshProposalsCount, refreshVotesCount } from '../src/helpers/actions';
+import db from '../src/helpers/mysql';
+
+// Usage: yarn ts-node scripts/generate_user_space_activities_counters.ts
+async function main() {
+  console.log('Inserting/Updating proposals_count');
+  const proposalsCountRes = await refreshProposalsCount();
+  console.log(proposalsCountRes);
+
+  console.log('Inserting/Updating votes_count');
+  let page = 0;
+  const batchSize = 1000;
+  const results: any[] = [];
+
+  while (true) {
+    const result = await db.queryAsync('SELECT id FROM spaces ORDER BY RAND () LIMIT ? OFFSET ?', [
```

Comment:
With `RAND` we may get the spaces that are already refreshed right?

Reply:
Is there a way to retrieve the spaces, sorted by votes_count ?

Reply:
edited to process one space at a time